docs/source/VUDRM-Speke-Key-Provider.md

```diff
@@ -1,6 +1,8 @@
 # SPEKE KEY PROVIDER API
 These are the docs for the speke key provider, which has a speke endpoint for use with AWS.
 
+In order to use our SPEKE API you will need to set up an API gateway that adds the header `API-KEY` with your API key and with your client name inplace of `client-name` in "https://speke.vudrm.tech/client-name/speke".
```

Comment:
⚠️ there is an and without a comma

Reply:
The oxford comma only applies to lists of 3 or more things so adding a comma here would be incorrect 😉